cvmfs/upload_s3.cc

```diff
@@ -225,6 +225,7 @@ void S3Uploader::FileUpload(
   UploadJobInfo(info);
   LogCvmfs(kLogUploadS3, kLogDebug, "Uploading from file finished: %s",
            local_path.c_str());
+  // CountUploadedBytes(???);
```

Comment:
I am not sure how to get the File size here.
GetFileSize(local_path) will probably return the uncompressed size of the file.

Reply:
I think you should use the compressed file size. The local path points to a compressed temporary file.

In any case, in `LocalUploader::FileUpload` you are also recording the compressed file size.